test.properties

```diff
@@ -4953,7 +4953,7 @@
     test.batch.dist.app.servers[dev-tools]=tomcat
 
     test.batch.names[dev-tools]=\
-        gradle-plugins-test-jdk8,\
+        #gradle-plugins-test-jdk8,\
```

Comment:
@kyle-miho we can disable these in a more granular fashion - this is too heavy.

Reply:
Actually you removed the exclusions here: 4716d9118fc2e. We can just re-add the ones we need.